backend/src/modules/users/users.entity.ts

```diff
@@ -0,0 +1,70 @@
+import { Column, CreateDateColumn, DeleteDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity({ name: 'users' })
+export class Users {
+  @PrimaryGeneratedColumn('uuid', {
+    name: 'user_id',
+  })
+  id: string;
+
+  @Column({
+    type: 'varchar',
+    length: 100,
+    nullable: false,
+  })
+  first_name: string;
```

Comment:
Should be camelCase, snake case should be in database, so you provice in decorator @Column new property name with value first_name